### ISSUE TITLE
Nima: ClientRequestImpl

### ISSUE THREAD

**Issue by tomas-langer** [helidon-io/helidon#5386]
HTTP/1.1 client `ClientRequestImpl` fix TODOs
@klustria please create follow up issues for some of the standalone bigger tasks (such as proxy support)

**Comment by klustria** [comment]
Spawned issue https://github.com/helidon-io/helidon/issues/6006 off of this specific for Proxy support

**Comment by klustria** [comment]
Currently blocked on this issue https://github.com/helidon-io/helidon/issues/6121 when doing Expect-100 on Transfer Encoding Chunk

**Comment by klustria** [comment]
PR https://github.com/helidon-io/helidon/pull/6208 is ready for review with changes that include the following:

- Configurable value for mediaContext, maxHeaderSize, maxStatusLength, sendExpect100Continue, and validateHeaders
- Optimize chunk request in ClientConnectionOutputStream, i.e. don’t chunk if data is small
- Send expect-100 (configurable via sendExpect100Continue) to verify host responsiveness before sending chunk request
- Configurable http connection queue limit
- Added read/connection timeouts in the connection queue key
- Request using HEAD method will be allowed only with no entity
- Mocked unit test and Integration test to validate code changes

**Comment by klustria** [comment]
PR https://github.com/helidon-io/helidon/pull/6208 was failing in io.helidon.nima.tests.integration.server.KeepAliveTest.sendWithKeepAliveExpectKeepAlive and the failure message is this: 
```
Error:  io.helidon.nima.tests.integration.server.KeepAliveTest.sendWithKeepAliveExpectKeepAlive  Time elapsed: 0.002 s  <<< ERROR!
[36719](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:36720)
java.io.UncheckedIOException: java.net.SocketException: Broken pipe
[36720](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:36721)
	at io.helidon.common.buffers.FixedBufferData.writeTo(FixedBufferData.java:74)
[36721](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:36722)
	at io.helidon.common.socket.PlainSocket.write(PlainSocket.java:127)
[36722](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:36723)
	at io.helidon.nima.webclient.http1.Http1ClientConnection$1.writeNow(Http1ClientConnection.java:168)
[36723](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:36724)
	at io.helidon.nima.webclient.http1.Http1ClientConnection$1.write(Http1ClientConnection.java:156)
[36724](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:36725)
	at io.helidon.nima.webclient.http1.ClientRequestImpl$ClientConnectionOutputStream.close(ClientRequestImpl.java:452)
[36725](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:36726)
	at io.helidon.nima.tests.integration.server.KeepAliveTest.lambda$testCall$2(KeepAliveTest.java:114)
[36726](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:36727)
	at io.helidon.nima.webclient.http1.ClientRequestImpl.outputStream(ClientRequestImpl.java:226)
[36727](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:36728)
	at io.helidon.nima.webclient.http1.ClientRequestImpl.outputStream(ClientRequestImpl.java:58)
[36728](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:36729)
	at io.helidon.nima.tests.integration.server.KeepAliveTest.testCall(KeepAliveTest.java:110)
[36729](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:36730)
	at io.helidon.nima.tests.integration.server.KeepAliveTest.sendWithKeepAliveExpectKeepAlive(KeepAliveTest.java:92)
[36730](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:36731)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
...
```

**Comment by klustria** [comment]
The failure also looks intermittent and this is the log from the pipeline:
```
[INFO] Results:
[36996](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:36997)
[INFO] 
[36997](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:36998)
Error:  Errors: 
[36998](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:36999)
Error:  io.helidon.nima.tests.integration.server.KeepAliveTest.sendWithKeepAliveExpectKeepAlive
[36999](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37000)
[INFO]   Run 1: PASS
[37000](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37001)
Error:    Run 2: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37001](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37002)
Error:    Run 3: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37002](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37003)
Error:    Run 4: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37003](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37004)
Error:    Run 5: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37004](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37005)
Error:    Run 6: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37005](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37006)
Error:    Run 7: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
Error:    Run 8: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37007](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37008)
Error:    Run 9: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37008](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37009)
Error:    Run 10: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37009](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37010)
Error:    Run 11: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37010](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37011)
Error:    Run 12: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37011](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37012)
Error:    Run 13: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37012](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37013)
Error:    Run 14: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37013](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37014)
Error:    Run 15: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37014](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37015)
Error:    Run 16: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37015](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37016)
Error:    Run 17: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37016](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37017)
Error:    Run 18: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37017](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37018)
Error:    Run 19: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37018](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37019)
Error:    Run 20: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37019](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37020)
Error:    Run 21: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37020](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37021)
Error:    Run 22: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37021](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37022)
Error:    Run 23: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37022](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37023)
Error:    Run 24: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37023](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37024)
Error:    Run 25: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37024](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37025)
Error:    Run 26: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37025](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37026)
Error:    Run 27: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37026](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37027)
Error:    Run 28: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37027](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37028)
Error:    Run 29: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37028](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37029)
Error:    Run 30: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37029](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37030)
Error:    Run 31: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37030](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37031)
Error:    Run 32: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37031](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37032)
Error:    Run 33: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37032](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37033)
Error:    Run 34: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37033](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37034)
Error:    Run 35: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37034](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37035)
Error:    Run 36: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37035](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37036)
Error:    Run 37: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37036](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37037)
Error:    Run 38: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37037](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37038)
Error:    Run 39: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
Error:    Run 40: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37039](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37040)
Error:    Run 41: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37040](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37041)
Error:    Run 42: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37041](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37042)
Error:    Run 43: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37042](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37043)
Error:    Run 44: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37043](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37044)
Error:    Run 45: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37044](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37045)
Error:    Run 46: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37045](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37046)
Error:    Run 47: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37046](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37047)
Error:    Run 48: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37047](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37048)
Error:    Run 49: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37048](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37049)
Error:    Run 50: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37049](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37050)
Error:    Run 51: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37050](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37051)
Error:    Run 52: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37051](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37052)
Error:    Run 53: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37052](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37053)
Error:    Run 54: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37053](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37054)
Error:    Run 55: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37054](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37055)
Error:    Run 56: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37055](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37056)
Error:    Run 57: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37056](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37057)
[INFO]   Run 58: PASS
[37057](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37058)
[INFO]   Run 59: PASS
[37058](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37059)
[INFO]   Run 60: PASS
[37059](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37060)
[INFO]   Run 61: PASS
[37060](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37061)
[INFO]   Run 62: PASS
[37061](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37062)
[INFO]   Run 63: PASS
[37062](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37063)
[INFO]   Run 64: PASS
[37063](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37064)
[INFO]   Run 65: PASS
[37064](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37065)
[INFO]   Run 66: PASS
[37065](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37066)
[INFO]   Run 67: PASS
[37066](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37067)
[INFO]   Run 68: PASS
[37067](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37068)
Error:    Run 69: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37068](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37069)
[INFO]   Run 70: PASS
[37069](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37070)
[INFO]   Run 71: PASS
[INFO]   Run 72: PASS
[37071](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37072)
[INFO]   Run 73: PASS
[37072](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37073)
Error:    Run 74: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37073](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37074)
[INFO]   Run 75: PASS
[37074](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37075)
[INFO]   Run 76: PASS
[37075](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37076)
[INFO]   Run 77: PASS
[37076](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37077)
[INFO]   Run 78: PASS
[37077](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37078)
[INFO]   Run 79: PASS
[37078](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37079)
[INFO]   Run 80: PASS
[37079](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37080)
[INFO]   Run 81: PASS
[37080](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37081)
[INFO]   Run 82: PASS
[37081](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37082)
[INFO]   Run 83: PASS
[37082](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37083)
[INFO]   Run 84: PASS
[37083](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37084)
[INFO]   Run 85: PASS
[37084](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37085)
[INFO]   Run 86: PASS
[37085](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37086)
[INFO]   Run 87: PASS
[37086](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37087)
[INFO]   Run 88: PASS
[37087](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37088)
[INFO]   Run 89: PASS
[37088](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37089)
Error:    Run 90: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37089](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37090)
Error:    Run 91: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37090](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37091)
Error:    Run 92: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37091](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37092)
Error:    Run 93: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37092](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37093)
Error:    Run 94: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37093](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37094)
Error:    Run 95: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37094](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37095)
Error:    Run 96: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37095](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37096)
Error:    Run 97: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37096](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37097)
Error:    Run 98: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37097](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37098)
Error:    Run 99: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37098](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37099)
Error:    Run 100: KeepAliveTest.sendWithKeepAliveExpectKeepAlive:92->testCall:110->lambda$testCall$2:114 » UncheckedIO
[37099](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37100)
[INFO] 
[37100](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37101)
[INFO] 
[37101](https://github.com/helidon-io/helidon/actions/runs/4277556279/jobs/7446445642#step:4:37102)
Error:  Tests run: 292, Failures: 0, Errors: 1, Skipped: 2
```

**Comment by klustria** [comment]
For now `sendExpect100Continue` is defaulted to false to avoid the above problem.

**Comment by klustria** [comment]
Spawned issue https://github.com/helidon-io/helidon/issues/6338 as a new task specific for validation of request headers in ClientRequestImpl.

**Comment by klustria** [comment]
PR https://github.com/helidon-io/helidon/pull/6208 is completed, hence closing. A separate issue needs to be opened with regards to io.helidon.nima.tests.integration.server.KeepAliveTest.sendWithKeepAliveExpectKeepAlive failing when sendExpect100Continue is set to true or defaulted to true.
